Update workshops-faq.md

danielle06 opened the pull request:
Remove curriculum section - will add information about requirements for COW curriculum in pre-workshop section


---

tracykteal commented:
@sheraaronhurt thanks for putting in the PR. Do they want to post this now, so we should merge?

sheraaronhurt commented:
@tracykteal , @kariljordan  said she would would merge next week. I'd prefer for this to go out on Monday so that it is fresh and not get lost over the weekend mail :)

kariljordan commented:
I fixed the filename. FYI all blog posts should include the date in the filename in this format: **YYYY-MM-DD-NAME-OF-BLOG-POST.md**

I changed the blog post date to today.

Great post @sheraaron!

danielle06 committed:
Update workshops-faq.md

Remove curriculum section - will add information about requirements for COW curriculum in pre-workshop section
